Puppetfile.lock

```diff
@@ -19,66 +19,54 @@ FORGE
       puppetlabs-stdlib (< 5.0.0, >= 4.2.0)
     katello-common (1.0.1)
       puppetlabs-stdlib (< 5.0.0, >= 4.2.0)
-    katello-foreman_proxy_content (6.0.0)
+    katello-foreman_proxy_content (6.1.1)
       katello-certs (< 4.0.0, >= 2.0.0)
       katello-pulp (< 6.0.0, >= 4.2.0)
-      katello-qpid (< 4.0.0, >= 3.0.0)
-      puppet-extlib (< 2.0.0, >= 0.10.4)
-      puppetlabs-apache (< 3.0.0, >= 1.0.0)
+      katello-qpid (< 5.0.0, >= 4.0.0)
+      puppet-extlib (< 3.0.0, >= 0.10.4)
+      puppetlabs-apache (< 3.0.0, >= 2.0.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.13.0)
-      theforeman-foreman_proxy (< 7.0.0, >= 5.0.0)
-    katello-katello (7.0.0)
-      katello-candlepin (< 5.0.0, >= 4.0.0)
-      katello-certs (< 4.0.0, >= 2.0.0)
-      katello-pulp (< 6.0.0, >= 4.3.0)
-      katello-qpid (< 4.0.0, >= 3.0.0)
-      puppet-extlib (< 2.0.0, >= 0.10.4)
+      theforeman-foreman_proxy (< 8.0.0, >= 5.0.0)
+    katello-katello (7.2.0)
+      katello-candlepin (< 6.0.0, >= 4.0.0)
+      katello-certs (< 5.0.0, >= 2.0.0)
+      katello-pulp (< 6.0.0, >= 5.3.0)
+      katello-qpid (< 5.0.0, >= 3.0.0)
+      puppet-extlib (< 3.0.0, >= 0.10.4)
       puppetlabs-apache (< 3.0.0, >= 1.0.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.6.0)
-      theforeman-foreman (< 9.0.0, >= 7.0.0)
-    katello-pulp (5.0.0)
-      katello-qpid (< 4.0.0, >= 2.0.0)
-      puppet-extlib (< 2.0.0, >= 0.10.4)
+      theforeman-foreman (< 10.0.0, >= 7.0.0)
+    katello-pulp (5.3.0)
+      katello-qpid (< 5.0.0, >= 2.0.0)
+      puppet-extlib (< 3.0.0, >= 0.10.4)
+      puppet-mongodb (< 3.0.0, >= 1.1.0)
       puppetlabs-apache (< 3.0.0, >= 1.2.0)
       puppetlabs-concat (< 5.0.0, >= 1.0.0)
-      puppetlabs-mongodb (< 2.0.0, >= 0.10.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.6.0)
       thias-squid3 (= 1.0.2)
-    katello-qpid (3.1.1)
+    katello-qpid (4.0.0)
       camptocamp-systemd (< 2.0.0, >= 0.4.0)
       puppetlabs-concat (< 5.0.0, >= 1.0.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.13.0)
     katello-service_wait (1.0.3)
       puppetlabs-stdlib (< 5.0.0, >= 4.2.0)
     puppet-extlib (1.1.0)
       puppetlabs-stdlib (>= 0)
-    puppet-staging (2.2.0)
-    puppetlabs-apache (2.3.0)
+    puppet-mongodb (2.0.0)
```

Comment:
Not sure about switching from puppetlabs-mongodb 1.x to puppet-mongodb 2.0. Our usage of mongodb parameters is minimal so it's probably not a problem but there is a breaking change in there.